D1673/P1673.md

```diff
@@ -5106,6 +5106,34 @@ void triangular_matrix_matrix_right_solve(
 }
 ```
 
+## General [linalg.general]
+
+[1]{.pnum} For the effects of all functions in this clause, when the effects
+are described as "Computes $R = EXPR$" or "compute $R = EXPR$" (for some $R$
+and mathematical expression $EXPR$), the following apply:
+
+  * [1.1]{.pnum} $EXPR$ has the conventional mathematical meaning as written.
+
+  * [1.2]{.pnum} The pattern $x^T$ should be read as "the transpose of $x$".
+
+  * [1.3]{.pnum} The pattern $x^H$ should be read as "the conjugate transpose of $x$".
+
+  * [1.4]{.pnum} When $R$ is the same name as a function parameter whose
+    type is a template parameter with `Out` in its name, the intent is that
+    the result of the computation is written to the elements of the function
+    parameter `R`.
+
+[2]{.pnum} For any function `F` that takes a `Triangle t` parameter, `t`
+applies to accesses to the previous parameter to the function.  Let `M` be
+such an access-modified function parameter.  `F` will only access the triangle
+of `M` specified by `t`.  For accesses `M[i, j]` in the domain of `M` but
+outside the triangle specified by `t`, `F` will interpret the value `M[i, j]`
+as equal to _`conj-if-needed`_`(M[j, i])` if the name of `F` starts with
+`hermitian`, and `M[j, i]` otherwise.
```

Comment:
We might want to call out that we only need the `M[j, i]` branch for `symmetric_*` functions.  The `triangle_` functions actually assume that `M[i, j]` is zero outside the specified triangle.  (We might also want to consider adding that here.)

Reply:
Ah, I didn't get that subtlety.  Yeah, you should definitely add that as a third case.

Reply:
Done!